src/partition/sph-partition-matches.cc

```diff
@@ -293,9 +302,13 @@ void Worker::defineOptions(po::options_description& opts) {
                        "chunk info is available for the identifier.");
     part.add_options()("part.chunk-mismatch-abort", po::bool_switch()->default_value(false),
```

Comment:
I've been curious about this option -- do you think it is useful to have?  Isn't the whole _point_ of the match partitioner to deal with this case?  Why would we want to abort?

Reply:
I added that option when debugging the input for the table `MatchesTrurth` in `DP02`. That was done (I think, on a different ticket). Perhaps, it's no longer needed. If you think the option is no longer needed then I can eliminate it.

Reply:
This option has been removed as it's no longer needed and to prevent confusing others.